roadmap/src/main/webapp/question.html

```diff
@@ -19,63 +19,113 @@
     <meta name="google-signin-client_id" content="1055777818620-l4sjm83glehidqtf7niespm6qq5rjeig.apps.googleusercontent.com.apps.googleusercontent.com">
   </head>
 <body class="bg question-banner">
-  <div class="mask" style="background-color: rgba(0,0,0,0.6)">
+  <div class="mask" style="height: 100%; background-color: rgba(0,0,0,0.6); overflow:scroll">
   <div class="row">
     <div class="col-12">
-      <div class="col-6 offset-3 white-bg">
+      <div class="col-6 offset-3 white-bg" style="height:104%; filter:drop-shadow(0 0 0.75rem #000);">
         <div class="col-8 offset-2 d-flex justify-content-center">
           <div class="mx-auto">
             <a href="/#"><img src="img/node-tree.svg" width=90 height=90 alt="Node Tree"></a>
           </div>
         </div>
-        <form method="POST" action="/data" style="position:relative; top:55px;">
+        <form method="POST" class="row g-3 needs-validation" name="signupForm" action="/tree-landing.html" onSubmit="return checkInput(this)" style="position:relative; top:55px;" novalidate>
```

Comment:
The "action= "/data" was connected to the the datastore servlet and by removing it, datastore wont work. How are you using action="/tree landing.html". I assumed that connected the submit button to the tree landing page which i took care of by putting a redirect to treelanding page on the servlet. Let me know if what I am saying makes sense.

Reply:
No you're right I unintentionally removed it
I was using `action` to forcibly push the user to the next page since I was having an issue with the buttons and their links. The change has been made